app/services/sms_notification_service.rb

```diff
@@ -29,7 +29,7 @@ def send_reminder_sms(reminder_type, appointment, callback_url, locale = DEFAULT
   def send_sms(body, callback_url = '')
     client.messages.create(
       from: sender_phone_number,
-      to: recipient_number.insert(0, I18n.t('sms.country_code')),
+      to: recipient_number.insert(0, ENV['SMS_COUNTRY_CODE'] || I18n.t('sms.country_code')),
```

Comment:
What do we think about moving this information to the `countries.rb` initializer?

Reply:
Great idea. We should do that now, as we're cold fixing 👍